python/tank/bootstrap/import_handler.py

```diff
@@ -74,6 +83,8 @@ def swap_core(cls, core_path):
             # Ignore all warnings
             warnings.simplefilter("ignore")
```

Comment:
I know it's not your code, but have we considered looking at
https://docs.python.org/2/library/warnings.html#available-context-managers? I always cringe when I see a try: finally: block when I know it could be solved with a from __future__ import with_statement at the top.